Author: Benoit Mortier
Description: put the Debian directory for scriptaculous into the script tag

Index: pkg/core/ihtml/themes/default/headers.tpl
===================================================================
--- pkg.orig/core/ihtml/themes/default/headers.tpl	2013-11-16 18:44:27.000000000 +0100
+++ pkg/core/ihtml/themes/default/headers.tpl	2013-11-16 18:47:18.145142151 +0100
@@ -15,14 +15,14 @@
 
   <link rel="shortcut icon" href="favicon.ico"/>
 
-  <script src="include/prototype.js" type="text/javascript"></script>
+  <script src="javascript/prototype/prototype.js" type="text/javascript"></script>
   <script src="include/fusiondirectory.js" type="text/javascript"></script>
 {if $usePrototype == 'true'}
-  <script src="include/scriptaculous.js" type="text/javascript"></script>
-  <script src="include/builder.js" type="text/javascript"></script>
-  <script src="include/effects.js" type="text/javascript"></script>
-  <script src="include/dragdrop.js" type="text/javascript"></script>
-  <script src="include/controls.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/scriptaculous.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/builder.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/effects.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/dragdrop.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/controls.js" type="text/javascript"></script>
   <script src="include/pulldown.js" type="text/javascript"></script>
   <script src="include/datepicker.js" type="text/javascript"></script>
 {/if}
